packages/event-normalizer/index.js

```diff
@@ -82,7 +82,9 @@ const events = {
   'aws:kafka': (event) => {
     for (const record in event.records) {
       for (const topic of event.records[record]) {
-        topic.value = base64Parse(topic.value)
+        if (topic.value) {
+          topic.value = base64Parse(topic.value)
+        }
```

Comment:
Can you update to: `topic.value ??= base64Parse(topic.value)`

Reply:
Added the test provided by @nikovirtala . Update to `topic.value ??= base64Parse(topic.value)` causes test to fail since base64Parse is run on empty value.

Reply:
Of course, `&&=` will ensure it runs when it exists